doc/src/modules/combinatorics/pc_groups.rst

```diff
@@ -1,12 +1,307 @@
+  polycyclic series(pc_series).
+
+* Computation of relative order for polycyclic series.
+
+* Implementation of class Collector which can be treated as a base for polycylic groups.
+
+* Implementation of polycyclic group presentation(pc_presentation for short).
+
+* Computation of exponent vector, depth and leading exponent for a given element
+  of a polycyclic group.
+
+For a description of fundamental algorithms of polycyclic groups, we
+often make use of *Handbook of Computational Group Theory*.
+
+
+The Construction of Polycyclic Groups
+-------------------------------------
+
+Given a Permutation Group, A Polycyclic Group is constructed by computing the
+corresponding polycylic generating sequence, polycyclic series and it's
+relative order.
+
+Attributes of PolycyclicGroup
+`````````````````````````````
+
+* ``pc_sequence`` : Polycyclic sequence is formed by collecting all the missing 
+  generators between the adjacent groups in the derived series of given
+  permutation group.
+
+* ``pc_series`` : Polycyclic series is formed by adding all the missing generators
+  of ``der[i+1]`` in ``der[i]``, where `der` represents derived series.
+
+* ``relative_order`` : A list, computed by the ratio of adjacent groups in pc_series.
+
+* ``collector`` : By default, it is None. Collector class provides the polycyclic presentation.
+
+>>> from sympy.combinatorics.named_groups import SymmetricGroup
+>>> G = SymmetricGroup(4)
+>>> PcGroup = G.polycyclic_group()
+>>> len(PcGroup.pcgs)
+4
+>>> pc_series = PcGroup.pc_series
+>>> pc_series[0]==G
+True
+>>> gen = pc_series[len(pc_series)-1].generators[0]
+>>> gen.is_identity
+True
+>>> PcGroup.relative_order
+[2, 3, 2, 2]
+
+
+The Construction of Collector
+-----------------------------
+
+Collector is one of the attributes of class PolycyclicGroup.
+
+Attributes of Collector
+```````````````````````
+
+Collector posses all the attributes of PolycyclicGroup, In addition there are
+few more attributes which are defined below:
+
+* ``free_group`` : free_group provides the mapping of polycyclic generating sequence with
+  the free group elements.
+
+* ``pc_presentation`` : Provides the presentation of polycyclic groups with the
+  help of power and conjugate relators.
+
+>>> from sympy.combinatorics.named_groups import SymmetricGroup
+>>> G = SymmetricGroup(3)
+>>> PcGroup = G.polycyclic_group()
+>>> Collector = PcGroup.collector
+>>> Collector.free_group
+<free group on the generators (x0, x1)>
+>>> Collector.pc_presentation
+{x0**2: (), x1**3: (), x0**-1*x1*x0: x1**2}
+
+
+Computation of Minimal Uncollected Subword
+``````````````````````````````````````````
+
+A word `V` defined on generators in the free_group of pc_group is a minimal
+uncollected subword of the word `W` if `V` is a subword of `W` and it has one of
+the following form:
+
+* `v = {x_{i+1}}^{a_j}x_i`
+* `v = {x_{i+1}}^{a_j}{x_i}^{-1}`
+* `v = {x_i}^{a_j}`
+
+`a_j \notin \{0, \ldots relative\_order[j]-1\}`.
```

Comment:
This could probably be `\text{relative\_order}[j]`.

Reply:
In `\text{relative\_order}` `\` before underscore is not working, it is printed as it is in html. So, I tried without any delimiter before underscore but that seems failing. I think we should try `\mathrm`.

Reply:
`\mathrm` should be ok if it works better. The font will probably be the same.

Reply:
Yes, font is same for both.